css-four-column-layout/styles.css

```diff
@@ -0,0 +1,64 @@
+* {
+  box-sizing: border-box;
+  font-family: barlow, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  max-width: 1500px;
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+}
+
+.columns {
+  display: flex;
+  margin: auto;
+  text-align: center;
+}
+
+.icon-container > img {
+  width: 50px;
+  justify-content: center;
+}
+
+.screen-container {
+  text-align: center;
+}
+
+.screen-container > img {
+  text-align: center;
+  width: 100%;
+}
+
+.icon-container {
+  text-align: center;
+}
+
+.column h3 {
+  text-align: center;
+  margin-bottom: 2rem;
+}
+
+.column p {
+  text-align: center;
+}
+
+.title {
+  text-align: center;
+  margin: auto;
+  flex-basis: 100%;
+  margin-bottom: 1rem;
+}
+
+.row {
+  display: flex;
+  flex-direction: row;
+}
+
+.card {
+  flex-basis: 25%;
+}
```

Comment:
This should be your `column` class. You should name it `column-fourth` to make its size clear.